2022-04-13 04:21:24,564 INFO    MainThread:3440 [wandb_setup.py:_flush():69] setting env: {}
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_setup.py:_flush():69] setting login settings: {}
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220413_042124-2bk9int3/logs/debug.log
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220413_042124-2bk9int3/logs/debug-internal.log
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_init.py:init():369] calling init triggers
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-04-13 04:21:24,565 INFO    MainThread:3440 [wandb_init.py:init():418] starting backend
2022-04-13 04:21:24,565 INFO    MainThread:3440 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-13 04:21:24,578 INFO    MainThread:3440 [backend.py:ensure_launched():123] starting backend process...
2022-04-13 04:21:24,589 INFO    MainThread:3440 [backend.py:ensure_launched():128] started backend process with pid: 3466
2022-04-13 04:21:24,594 INFO    MainThread:3440 [wandb_init.py:init():423] backend started and connected
2022-04-13 04:21:24,596 INFO    MainThread:3440 [wandb_init.py:init():465] updated telemetry
2022-04-13 04:21:24,596 INFO    MainThread:3440 [wandb_init.py:init():484] communicating current version
2022-04-13 04:21:26,214 INFO    MainThread:3440 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-13 04:21:26,215 INFO    MainThread:3440 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-04-13 04:21:26,616 INFO    MainThread:3440 [wandb_init.py:init():522] starting run threads in backend
2022-04-13 04:21:29,342 INFO    MainThread:3440 [wandb_run.py:_console_start():1524] atexit reg
2022-04-13 04:21:29,343 INFO    MainThread:3440 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-04-13 04:21:29,345 INFO    MainThread:3440 [wandb_run.py:_redirect():1376] Redirecting console.
2022-04-13 04:21:29,347 INFO    MainThread:3440 [wandb_run.py:_redirect():1438] Redirects installed.
2022-04-13 04:21:29,348 INFO    MainThread:3440 [wandb_init.py:init():546] run started, returning control to user process
2022-04-13 04:21:29,651 INFO    MainThread:3440 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-04-13 04:21:29,653 INFO    MainThread:3440 [wandb_run.py:_restore():1466] restore
2022-04-13 04:21:32,455 INFO    MainThread:3440 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 1046
}

2022-04-13 04:21:32,640 INFO    MainThread:3440 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 7084
}

